CMakeLists.txt

```diff
@@ -312,6 +312,8 @@ endif()
 # dependencies.
 add_library(umf_headers INTERFACE)
 
+set(CMAKE_INSTALL_RPATH "\$ORIGIN")
```

Comment:
I think it requires at least a small-ish comment here 😉

Reply:
Added a comment